src/main/java/com/mojang/brigadier/tree/CommandNode.java

```diff
@@ -137,22 +204,70 @@ public int hashCode() {
         return 31 * children.hashCode() + (command != null ? command.hashCode() : 0);
     }
 
+    /**
+     * Returns the requirement for this command, that is used by {@link #canUse(Object)}.
+     *
+     * @return the requirement for this command, that is used by {@link #canUse(Object)}
+     */
     public Predicate<S> getRequirement() {
         return requirement;
     }
 
+    /**
+     * Returns the name of this command.
```

Comment:
"command" -> "command node" / "node"; same for following other methods of this class